product-service/src/main/java/com/yandex/market/productservice/controller/SellerController.java

```diff
@@ -52,7 +53,7 @@ public Page<SellerProductPreview> findProductsBySellerId(
         return productService.getProductsBySellerId(sellerId, pageable);
     }
 
-    @GetMapping("{sellerId}/archived-products")
+    @GetMapping("/sellers/{sellerId}/products/archived-products")
```

Comment:
Мне кажется тут лишний /products/

```suggestion
    @GetMapping("/sellers/{sellerId}/archived-products")
```

Reply:
по логике просто архив не имеет связи со списком продуктов

Reply:
Это параллелная списку продуктов действие, а не вытекающее из неё